dev/fattest.simplicity/src/componenttest/topology/impl/LibertyClient.java

```diff
@@ -623,6 +633,11 @@ public ProgramOutput startClientWithArgs(boolean preClean, boolean cleanStart,
                 LOG.warning("The build is configured to run FAT tests with Java 2 Security enabled, but the FAT client " + getClientName() +
                             " is exempt from Java 2 Security regression testing.");
             }
+
+            // If we are running on Java 18+, then we need to explicitly enable the security manager
+            if (javaVersion() >= 18) {
+                JVM_ARGS += " -Djava.security.manager=allow";
+            }
```

Comment:
This logic needs to be moved into the if above.  It should not be set in the else condition.

Reply:
Similarly to LibertyServer, you have access to JavaInfo in this method so you should be able to use it.  i.e. if (javaInfo.majorVersion() >= 18)

Reply:
There isn't a JavaInfo.forClient() method like there is JavaInfo.forServer(), is JavaInfo.forCurrentVM() sufficient, or should I implement a forClient() method on JavaInfo?

Reply:
javaInfo is JavaInfo.forCurrentVM() which should be fine.